tests/monitoring/test_non_theano_monitoring.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+import theano
+from fuel.datasets import IterableDataset
+from numpy.testing import assert_raises
+
+from blocks.graph import ComputationGraph
+from blocks.monitoring.evaluators import DatasetEvaluator
+from tests.monitoring.test_aggregation import TestBrick
+from blocks.bricks.cost import CategoricalCrossEntropy
+from blocks.extensions.monitoring import CrossEntropy
+
+floatX = theano.config.floatX
+
+
+def test_dataset_evaluators():
+    X = theano.tensor.vector('X')
+    Y = theano.tensor.vector('Y')
+
+
+    data = [numpy.arange(1, 4, dtype=floatX).reshape(3, 1) / 7.0,
+            numpy.arange(11, 14, dtype=floatX).reshape(3, 1) / 17.0]
+
```

Comment:
It's 3, because the first axis is the batch-axis and you're reshaping to (3, 1) i.e. 3 batches with 1 element.